aiida/orm/nodes/data/code.py

```diff
@@ -494,6 +494,8 @@ def get_full_text_info(self, verbose=False):
         """
         Return a (multiline) string with a human-readable detailed information on this computer
```

Comment:
The docstring also seems outdated - the function returns a nested list of strings, not a multiline string.

Reply:
Thanks, have updated it.